.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: php
+
+php:
+  - "7.2"
+
+services:
+  - docker
+
+install: make install
```

Comment:
for the record: I'm not sure if the `make install` installs the php dependencies...